indico/htdocs/js/indico/modules/registration/registration.js

```diff
@@ -47,4 +48,154 @@
             $('#payment-summary').effect('highlight', 800);
         });
     }
+
+    function setupStaticURLGeneration() {
+        $('.js-static-url').on('click', function(event) {
+            var $this = $(this);
+            $.ajax({
+                method: 'POST',
+                url: $this.data('href'),
+                success: function(result) {
+                    var content = $('.registrations > .clipboard-dialog').clone();
+                    content.find('input:text').val(result.url);
+
+                    $this.qtip({
+                        content: {
+                            text: content
+                        },
+                        position: {
+                            my: 'top center',
+                            at: 'bottom center'
+                        },
+                        hide: {
+                            event: 'mouseleave',
+                            fixed: true,
+                            delay: 700
+                        },
+                        show: {
+                            event: false,
+                            ready: true
+                        },
+                        events: {
+                            show: function() {
+                                var tip = $(this);
+                                _.defer(function() {
+                                    tip.find('input:text').focus().select();
+                                });
+                            }
+                        }
+                    });
+                }
+            });
+        });
+    }
+
+    function setupTableSorter() {
+        $('.registrations .tablesorter').tablesorter({
+            cssAsc: 'header-sort-asc',
+            cssDesc: 'header-sort-desc',
+            headers: {
+                0: {
+                    sorter: false
+                }
+            }
+        });
+    }
+
+    function setupRegistrationsList() {
+        colorizeSelectedRows();
+        setupStaticURLGeneration();
+        setupTableSorter();
+
+        $('.registrations .toolbar')
+        .dropdown()
+        .on('click', '.js-dialog-action', function(e) {
+            e.preventDefault();
+            var $this = $(this);
+            ajaxDialog({
+                dialogClasses: 'reglist-filter-dialog',
+                trigger: this,
+                url: $this.data('href'),
+                title: $this.data('title'),
+                onClose: function(data) {
+                    if (data) {
+                        $('.registrations-table-wrapper').html(data.registrations_list);
```

Comment:
`registration_list` (I don't think using the plural form is correct in this context)